src/CodeGenerator/AbstractInjector.php

```diff
@@ -19,32 +19,24 @@
  */
 abstract class AbstractInjector implements InjectorInterface
 {
-    /**
-     * @var string[]|FactoryInterface[]
-     */
+    /** @var string[]|FactoryInterface[] */
     protected $factories = [];
 
-    /**
-     * @var FactoryInterface[]
-     */
+    /** @var FactoryInterface[] */
     private $factoryInstances = [];
 
-    /**
-     * @var ContainerInterface
-     */
+    /** @var ContainerInterface */
     private $container;
 
-    /**
-     * @var InjectorInterface
-     */
+    /** @var InjectorInterface */
     private $injector;
 
     /**
      * {@inheritDoc}
      */
-    public function __construct(InjectorInterface $injector, ContainerInterface $container = null)
+    public function __construct(InjectorInterface $injector, ?ContainerInterface $container = null)
```

Comment:
Please revert his change as it introduces BC break.

Reply:
These additional ? were added by the phpcbf, but I added the others Typehints. As soon as #15 is merged I will revert some of the Typehints that are breaking changes.

Reply:
Are you sure that this breaks BC? This is something the coding standard dictates. Afaik these types are already nullable   since they declare `null` as default value.

Reply:
@gennadigennadigennadi right, in `__construct` thats not an issue (as it is treated different to other methods).

@tux-rampage nope, not sure. Actually, `__construct` signature changes do not have implications as tested [here](https://3v4l.org/bW2O8)

Reply:
@boesing your example already leads to an error with the current codebase because it declares `null` as param default: https://3v4l.org/6GSPb

So `foo(type $var = null)` is an implicit `foo(?type = null)`. The coding style just enforces to make this obvious. So as far as I can tell this does not break BC or am I missing something else?

Reply:
I've re-checked this again and you are right... adding the nullable-flag to a typehint is not a BC break. I probably miss-understood something... Damn!

@gennadigennadigennadi sorry for bothering you with this.